cmd/frontend/graphqlbackend/own.go

```diff
@@ -137,6 +140,17 @@ type DeleteCodeownersFilesInput struct {
 	RepoName *string
 }
 
+type AssignOwnerArgs struct {
+	Input AssignOwnerInput
+}
+
+type AssignOwnerInput struct {
+	AssignedOwnerID   graphql.ID
```

Comment:
That's user ID, right?

```suggestion
	AssignedOwnerID   graphql.ID  // user ID
```

Reply:
yep